server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/SetMailboxes.feature

```diff
@@ -0,0 +1,13 @@
+Feature: SetMailboxes method
+
+Scenario: Renaming a mailbox should keep messages
```

Comment:
A scenario is a subelement of a feature, it's usually indented.
See https://github.com/cucumber/cucumber/wiki/Gherkin